cms/envs/common.py

```diff
@@ -571,27 +571,35 @@
 
 LOGGING_ENV = 'sandbox'
 
+# Public domain name of Studio (should be resolvable from the end-user's browser)
+CMS_BASE = 'localhost:18010'
+CMS_ROOT_URL = '//localhost:18010'
+
 LMS_BASE = 'localhost:18000'
 LMS_ROOT_URL = "https://localhost:18000"
 LMS_INTERNAL_ROOT_URL = LMS_ROOT_URL
 
 LOGIN_REDIRECT_URL = EDX_ROOT_URL + '/home/'
 # TODO: Determine if LOGIN_URL could be set to the FRONTEND_LOGIN_URL value instead.
-LOGIN_URL = reverse_lazy('login_redirect_to_lms')
-FRONTEND_LOGIN_URL = lambda settings: settings.LMS_ROOT_URL + '/login'
+# Original 'LOGIN_URL' renamed to 'LMS_REDIRECT_LOGIN_URL'
+# This is a candidate for conditional setings to switch between Studio local
+# login and LMS redirection login
+LMS_REDIRECT_LOGIN_URL = reverse_lazy('login_redirect_to_lms')
+LOGIN_URL = reverse_lazy('login')
+FRONTEND_LOGIN_URL = lambda settings: settings.CMS_ROOT_URL + '/login'
```

Comment:
Reduce the change with clearer `# Tahoe: ` comments so it's more obvious during release merge:

```suggestion
LOGOUT_REDIRECT_URL = reverse_lazy('home')  # Tahoe: To make `TAHOE_STUDIO_LOGIN` feature work
LOGIN_URL = reverse_lazy('login')  # Tahoe: To make `TAHOE_STUDIO_LOGIN` feature work

FRONTEND_LOGIN_URL = lambda settings: settings.LMS_ROOT_URL + '/login'
```